source/_patterns/_fonts.scss

```diff
@@ -1,65 +1,161 @@
 /*! URHEBERRECHTSHINWEIS: Die SRG SSR Schrift und die dazugehoerige Software stehen im vollumfaenglichen Eigentum der SRG. Jede Bearbeitung, Vervielfaeltigung, Verbreitung und/oder oeffentliche Wiedergabe, insbesondere die ungenehmigte Uebernahme der SRG SSR Schrift stellt einen Rechtsverstoss dar, der zum Schadenersatz verpflichtet und auch strafrechtlich verfolgt wird. */
 
 @font-face {
-  font-family: 'SRG_SSR_Regular';
-  src: url('../fonts/SRGSSRType_W_Rg.woff2') format('woff2'), url('../fonts/SRGSSRType_W_Rg.woff') format('woff');
-  font-weight: normal;
-  font-style: normal;
-  font-display: swap;
+  font-family: 'SRG_SSR_Display_Light';
+  src: url('../fonts/SRGSSRTypeDisplay_W_Lt.woff2') format('woff2'), url('../fonts/SRGSSRTypeDisplay_W_Lt.woff') format('woff');
+  font-weight: normal !important;
+  font-style: normal !important;
+  letter-spacing: swap;
```

Comment:
gibt's das? hab ich in keiner spec gefunden

Reply:
ou ja sollte font-display sein